libs/template/templates/default-sql/template/{{.project_name}}/.vscode/settings.json.tmpl

```diff
@@ -0,0 +1,28 @@
+{
+    "python.analysis.stubPath": ".vscode",
+    "databricks.python.envFile": "${workspaceFolder}/.env",
+    "jupyter.interactiveWindow.cellMarker.codeRegex": "^# COMMAND ----------|^# Databricks notebook source|^(#\\s*%%|#\\s*\\<codecell\\>|#\\s*In\\[\\d*?\\]|#\\s*In\\[ \\])",
+    "jupyter.interactiveWindow.cellMarker.default": "# COMMAND ----------",
+    "python.testing.pytestArgs": [
+        "."
+    ],
+    "python.testing.unittestEnabled": false,
+    "python.testing.pytestEnabled": true,
+    "python.analysis.extraPaths": ["src"],
+    "files.exclude": {
+        "**/*.egg-info": true,
+        "**/__pycache__": true,
+        ".pytest_cache": true,
+    },
+    "sqltools.connections": [
+        {
+            "connectionMethod": "VS Code Extension (beta)",
+            "catalog": "hive_metastore",
```

Comment:
Catalog needs to be templatized. 

We could also configure the schema:
pro: the users have easy access to all their tables and views
con: The users can't browse tables and views outside of their schema.

As a compromise, we could create to connections. One for all tables and one scoped down to the project.

Reply:
![Screenshot 2024-01-26 at 17 28 13](https://github.com/databricks/cli/assets/40952/7f93a928-5c2b-4eea-a34b-66765b74bbd6)

Reply:
```
    "sqltools.connections": [
        {
            "connectionMethod": "VS Code Extension (beta)",
            "catalog": "hive_metastore",
            "previewLimit": 50,
            "driver": "Databricks",
            "name": "Metastore",
            "group": "Databricks",
            "path": "/sql/1.0/warehouses/12345"
        },
        {
            "connectionMethod": "VS Code Extension (beta)",
            "catalog": "{{catalog}}",
            "previewLimit": 50,
            "driver": "Databricks",
            "name": "Project",
            "path": "/sql/1.0/warehouses/12345",
            "schema": "{{schema}}",
            "group": "Databricks"
        }
    ],
```